.gitignore

```diff
@@ -9,6 +9,9 @@ yarn-error.log*
 *.seed
 *.pid.lock
 
+# Visual Studio Code
+.vscode
```

Comment:
Should we exclude these folders entirely?  Currently we have added some **.vscode/launch.json** files to Git as templates.

Reply:
Even though I don't feel very strongly about this change, launch.json shows up as modified every time someone changes some local debug config - which is quite an inconvenience. How about we explicitly mark template folders as include in .gitignore?

Reply:
I'm in favor of ignoring.